### ISSUE TITLE
Response Graph not generated

### ISSUE THREAD

**Issue by PythonSmall-Q** [upptime/upptime#959]
**Describe the bug**
After I made a genaration of the project, even after 3 weeks I still can't get any type of Response graph from the last three actions in the pic below
![image](https://github.com/upptime/upptime/assets/106425289/fb3a0a7b-d27f-4227-ae16-0fbe42dfef16)

**Expected behavior**
every pic should be shown

**Screenshots**
![image](https://github.com/upptime/upptime/assets/106425289/eb8f98d2-fd69-410d-9944-2673f2b2516c)

**Repo**

https://github.com/XMOJ-Script-dev/statuspage

**Comment by AnandChowdhary** [comment]
I think it's because how the slugify works and it doesn't work well well on non-Latin script characters. If you add a `slug:` after the name of the site, it should work as expected.

**Comment by PythonSmall-Q** [comment]
but i used a slug, please see the repo thx

**Comment by PythonSmall-Q** [comment]
![image](https://github.com/upptime/upptime/assets/106425289/64ee665a-1844-4f69-a92f-73064e3ef7a9)

**Comment by PythonSmall-Q** [comment]
![image](https://github.com/upptime/upptime/assets/106425289/5f17395d-4b4c-4764-8775-d0d5213ee004)
now all graphs are missing

**Comment by PythonSmall-Q** [comment]
and the statuspage seem to be down because of this

**Comment by PythonSmall-Q** [comment]
@AnandChowdhary any updates？

**Comment by AnandChowdhary** [comment]
You're right, this is a bug where some places it's not correctly using the slug.

**Comment by PythonSmall-Q** [comment]
Gotta, looking around waiting for a fix

**Comment by SJang1** [comment]
Same issue here and some reason it went way wrong
![image](https://github.com/upptime/upptime/assets/29011440/dcf5a02d-e014-4f99-9090-17101e2935a4)
